sass/layout/_sidebar-left.scss

```
@@ -854,22 +858,27 @@
 
 
     .SidebarChannel.animating {
-        transition-property: height, padding, color;
-        transition-duration: 0.15s; /* collapse animation speed */
-        transition-timing-function: ease-out;
+        transition-property: height, padding, color, opacity, visibility;
+        transition-duration: 0.15s, 0.15s, 0.15s, 0.15s, 0.15s; /* collapse animation speed */
+        transition-timing-function: ease-out, ease-out, ease-out, ease-in, step-start;
```

Sorry, I wasn't clear here, this is what I meant.

```suggestion
        transition-timing-function: ease, ease, ease, ease-in, step-start;
```